docs/integrations/a11y.md

```diff
@@ -105,6 +105,15 @@ APCA is still a draft, and not part of WCAG 3. But APCA _is_ [well-researched](h
 
 The [APCA contrast algorithm](https://www.myndex.com/APCA/) is still in beta, but is a likely candidate for the upcoming WCAG 3 contrast algorithm. APCA uses the [latest research in color science](https://git.apcacontrast.com/documentation/WhyAPCA) and implements more perceptual contrast ratios that yield more accurate results. But it can be tricky to understand.
 
+| Setting         | Description                                                                      |
+| :-------------- | :------------------------------------------------------------------------------- |
+| `'bronze'`      | Enforce [Bronze](https://www.myndex.com/APCA/#Bronze)-level conformance.         |
+| `'bronze-body'` | Enforce Bronze-level compliance for body text (must hit `15 Lc` higher contrast) |
```

Comment:
🤔 I might be confused… I might be implementing `'silver'`, actually.

Pending review from the great Myndex himself